modules/EventEmitter.js

```diff
@@ -84,6 +100,7 @@ class EventEmitter {
                 data.response.send(res);
             }).catch((err) => {
                 logger.error(`Failed to get root hash for query ${data.query}`);
+                data.response.status(500);
                 data.response.send(500); // TODO rethink about status codes
```

Comment:
Would be super useful to see 
                 data.response.send({
                    status: 500,
                     message: 'Reason what failed so that user has some clue',
                 });